docs/usage_guide.rst

```diff
@@ -865,7 +865,7 @@ An xpub does not allow spending your coins but provides information about your w
 ENS names resolution
 =====================
 
-Rotki automatically resolves ens name for each of your evm accounts. If there is a primary ens name specified for an account, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
+Rotki automatically resolves ens name and ens avatar for every evm address that has ens name set in ethereum mainnet. If there is a primary ens name specified for an address, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name. If an ens avatar is set, it will be shown instead of the default address icon.
```

Comment:
```suggestion
Rotki automatically resolves ens name and ens avatar for every evm address that has ens name set in ethereum mainnet. If there is a primary ens name specified for an address, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name. If an ens avatar is set, it will be shown instead of the address blockie.
```